mandatory/4-tax.js

```diff
@@ -5,8 +5,13 @@
   Sales tax is 20% of the price of the product.
 */
 
+
 function calculateSalesTax() {}
 
+ function calculateSalesTax(sales) {
+   const calcTax = sales * 0.2 + sales;
+   return calcTax;
+ }
```

Comment:
You're declaring the same function name twice here, which is why one of your tests is failing. You can remove the declaration on line 9.